### DISCUSSION TITLE
Difference feature importance sklearn/shapash

### DISCUSSION THREAD

**Discussion by Redflyo** [MAIF/shapash#363] [Q&A] [answered]
Dear all,

I would like to know if it's normal to have difference between "features importances" sklearn and "features importances" shapash I use a Random Forest model. 
link of doc sklearn:
[https://scikit-learn.org/stable/auto_examples/ensemble/plot_forest_importances.html](https://scikit-learn.org/stable/auto_examples/ensemble/plot_forest_importances.html)

Thanks for all !

**Comment by ThomasBouche** [comment, marked as answer]
Hi, 
Sorry for the delay in responding.
In Shapash, feature importance is computed from the sum of the local contributions (compiled by shap by default). 
Feature importance is therefore the proportion of the global contributions of the feature.

So yes, the difference is normal between Shapash and Sklearn feature importance
